.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - checkout
       - run: npm install
-      - run: npm test
+      - run: npm test -- --no-watch
```

Comment:
you should probably be runnning `npm run circleci` command instead (better yet, the naming should be more like `test:circleci` or `test:ci`). Furthermore, you are completely missing `run` word. 
`npm <command>` runs some command related to NPM
`npm run <command>` runs a command from `package.json`

Reply:
Ok, thank you I didn't know that, it's fixed